data/en/qpses_0001.json

```diff
@@ -0,0 +1,816 @@
+                                "type": "Number",
+                                "label": "Full-time equivalent temporary and casual female employees",
+                                "description": "",
+                                "q_code": "662",
+                                "min_value": {
+                                    "answer_id": "ft-female-temp-casual-employees",
+                                    "exclusive": false
+                                },
+                                "decimal_places": 2
+                            }]
+                        }]
+                    },
+                    {
+                        "id": "total-emp-block",
+                        "type": "Question",
+                        "questions": [{
+                            "id": "total-emp-question",
+                            "title": "On {{ metadata['date_ref'] | format_date }}, what was {{ first_non_empty_item(metadata['trad_as'], metadata['ru_name']) }}&apos;s total number of employees?",
+                            "description": "",
+                            "guidance": {
+                                "content": [{
+                                    "description": "Include permanent, temporary, and casual employees."
+                                }]
+                            },
+                            "type": "General",
+                            "answers": [{
+                                "id": "total-number-of-emps",
+                                "mandatory": true,
+                                "type": "Number",
+                                "label": "Total number of employees",
+                                "description": "",
+                                "q_code": "50",
+                                "min_value": {
+                                    "value": 0,
+                                    "exclusive": true
+                                },
+                                "decimal_places": 0
+                            }]
+                        }]
+                    },
+                    {
+                        "id": "total-fte-block",
+                        "type": "Question",
+                        "questions": [{
+                            "id": "total-fte-question",
+                            "title": "On {{ metadata['date_ref'] | format_date }}, what was {{ first_non_empty_item(metadata['trad_as'], metadata['ru_name']) }}&apos;s total number of full-time equivalent employees?",
+                            "description": "<p> Please convert part-time employees hours into those worked by full-time employees. For example, if a part-time employee works 10 hours per week and the full-time working week in your organisation is 37 hours, the part-time employee would equate to 0.27 full-time equivalents (10 divided by 37). </p><p>Please use weekly contracted hours as the basis to calculate FTE.</p>",
```

Comment:
There is a leading space here in the `<p>` and there is also a description here that does not exist on the slides.